testsuite/integration-arquillian/tests/other/console/pom.xml

```diff
@@ -32,7 +32,8 @@
     <name>Admin Console UI Tests</name>
 
     <properties>
-        <keycloak.theme.dir>${auth.server.home}/themes</keycloak.theme.dir>
+        <keycloak.theme.dir>${auth.server.home}/themes-primary</keycloak.theme.dir>
+        <developer.theme.dir>${project.basedir}/../../../../../themes/src/main/resources/theme</developer.theme.dir>
```

Comment:
Ah.  I thought you meant I was somehow changing which server instance was being tested.  But in this case I don't understand your objection.   We have plenty of other tests that use the default configuration.

If you like, I can just move those two lines to a different profile.